test/dataset_collection/test_dataset_collection.py

```diff
@@ -18,165 +18,156 @@
 class TestDatasetCollection(TestCase):
     current_directory = Path(__file__).absolute().parent
 
-    def test_determine_type_without_return_without_arguments(self):
+    def setUp(self):
         filepath = self.current_directory / "Example.java"
-        ast = AST.build_from_javalang(build_ast(filepath))
+        self.ast = AST.build_from_javalang(build_ast(filepath))
```

Comment:
we do not have a general ast tree, it depends on a java class. It's a wrong method. Have the tests passed?

Reply:
i can rename it to example_ast. yes the tests passed